website/docs/reference/source-properties.md

```diff
@@ -25,6 +25,10 @@ sources:
     [loaded_at_field](resource-properties/freshness#loaded_at_field): <column_name>
     [meta](meta): {<dictionary>}
     [tags](resource-configs/tags): [<string>]
+    
+    # requires v1.1+
+    [config](resource-properties/config):
+      [<source_config>](source-configs): <config_value>
```

Comment:
I wasn't sure of the best move here. We can't magically combine `<VersionBlock>` and `<File>` components. It also feels wasteful to copy-paste the full contents, when it's such a small addition.

Reply:
I think Jason might be updating this in another PR. See his comments: https://github.com/dbt-labs/docs.getdbt.com/pull/1320/files#r849795638

Reply:
I think we need to leave this as is. I checked with @JKarlavige and it seems like there's no way to do this right now unless we version the entire block.